### ISSUE TITLE
needs requirements.txt

### ISSUE THREAD

**Issue by jayendra-ram** [hcoxec/h#1]

**Comment by hcoxec** [comment]
yep! as noted in the readme this the initial release, additional documentation including requirements and package info will be pushed this week. I'll update/close this issue when that happens.
